src/pages/bulkImport/BulkReportForm.jsx

```diff
@@ -91,6 +140,47 @@ export default function BulkReportForm({ assetReferences }) {
   const sightingFieldSchemas = useSightingFieldSchemas();
   const encounterFieldSchemas = useEncounterFieldSchemas();
 
+  const multiselectFieldIds = reduce(
+    [...sightingFieldSchemas, ...encounterFieldSchemas],
+    (memo, currentSchema) => {
+      if (currentSchema?.fieldType === 'multiselect')
+        return [...memo, currentSchema?.id];
+      return memo;
+    },
+    [],
+  );
+
+  const matchingCustomMultiselects = filter(
+    availableFields,
+    field => {
+      const matches = field?.key?.match(
+        /custom-(encounter|sighting|individual)-(.*)/,
+      ); // I imagine that there's a way to generalize this to work with deriveCustomFieldPrefix, but I played around with new regExp() using variables, and couldn't seem to get it to work. Would love some help with this.
```

Comment:
availableFields did not have customField and fieldType properties... likely because FlatFile does not like extraneous properties. 
Since their presence greatly simplified the readability of this code, I decided to add them to availableFields and then remove them in BulkReportForm after using them to filter the customMultiselects..., which happens in generateCustomMultiselectFlatFileHooks, which now lives in bulkImportFormatters. Change in https://github.com/WildMeOrg/codex-frontend/pull/431/commits/a7731152064b872d4adc78e8b19db537121fba6e.